src/base/abci/abc.c

```diff
@@ -8025,7 +8025,7 @@ int Abc_CommandExact( Abc_Frame_t * pAbc, int argc, char ** argv )
     if ( argc == globalUtilOptind )
         goto usage;
 
-    memset( pTruth, 0, 64 );
+    memset( pTruth, 0, 64 * sizeof(word) );
```

Comment:
I also got a warning for this while trying out gcc-9. However, is it more idiomatic with `64 * sizeof(pThruth[0])`?
Of course, the `64` is not nice in itself, but one thing at the time :)